test/api/dependency/samples.go

```diff
@@ -0,0 +1,54 @@
+package dependency
+
+import (
+	"github.com/konveyor/tackle2-hub/api"
+)
+
+// Set of valid resources for tests and reuse.
+type TestCase struct {
+	Name            string
+	ApplicationFrom api.Application
+	ApplicationTo   api.Application
+}
+
+var (
+	dependency = api.Dependency{
```

Comment:
Some of these lines seems to be redundant to me (the dependency here and init() below). But I'd suggest get it to (partially) working state (passing tests against locally running Hub https://github.com/konveyor/tackle2-hub/tree/main/test#rest-api) and we can refactor it then.

Reply:
Refactored the code Kindly review